core/graph/src/main/java/uk/gov/gchq/gaffer/graph/hook/NamedOperationResolver.java

```diff
@@ -92,6 +97,8 @@ private List<Operation> resolveNamedOperation(final NamedOperation namedOp, fina
             namedOpDetail = cache.getNamedOperation(namedOp.getOperationName(), user);
         } catch (final CacheOperationException e) {
             // Unable to find named operation - just return the original named operation
+            // Exception messages are lost so Log them
+            LOGGER.error(e.getMessage());
```

Comment:
Could we have something to explain this a bit more. E.g. "Unable to find Named Operation in cache with exception: {}". Otherwise the logs will contain the cache exception without any context.